app/ide-desktop/lib/types/modules.d.ts

```diff
@@ -2,6 +2,19 @@
  *
  * This file MUST NOT `export {}` for the modules to be visible to other files. */
 
+// Required because this is a build artifact, which does not exist on a clean repository.
+declare module '*/build.json' {
+    interface BuildInfo {
+        commit: string
+        version: string
+        engineVersion: string
+        name: string
+    }
```

Comment:
Isn't this duplicating the definition from [app/ide-desktop/lib/types/globals.d.ts](https://github.com/enso-org/enso/blob/9c15ba06bf22bed36fc69110c5b375699858d2c7/app/ide-desktop/lib/types/globals.d.ts#L22)? Can we somehow avoid this?

Reply:
good catch, i think the interfaces in `globals.d.ts` are globally accessible so i should be able to just remove the definition

Reply:
maybe not... should be able to import it though